hack/env.sh

```diff
@@ -35,3 +35,4 @@ export ENABLE_ADMISSION_CONTROLLER=${ENABLE_ADMISSION_CONTROLLER:-"true"}
 export CLUSTER_TYPE=${CLUSTER_TYPE:-openshift}
 export NAMESPACE=${NAMESPACE:-"openshift-sriov-network-operator"}
 export WEBHOOK_CA_BUNDLE=${WEBHOOK_CA_BUNDLE:-""}
+export DEV_MODE="FALSE"
```

Comment:
to allow to overwrite you can 👍 
export DEV_MODE=${DEV_MODE:-"FALSE"}

Reply:
Agreed